lib/getters/creditCardPaymentGetters.ts

```diff
@@ -18,21 +18,43 @@ const getPaymentType = (selectedType: string) => {
   return paymentType
 }
 
+const getExpireDate = (cardDetails): String =>
+  cardDetails?.expireMonth && cardDetails?.expireYear
+    ? cardDetails?.expireMonth + "/" + cardDetails?.expireYear
+    : ""
+
+const getCardEndingDigits = (cardNumber): String => (cardNumber ? "x" + cardNumber?.slice(-4) : "")
```

Comment:
getCardEndingDigits should take the cardDetails as a argument, not the number/string

Reply:
Resolved.